Editor/MaterialPropertyReorderableList.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections;
+using UnityEditor;
+using UnityEditorInternal;
+using UnityEngine;
+
+namespace Unity.MaterialSwitch
+{
+    public class MaterialPropertyReorderableList : ReorderableList
+    {
+        private string _filterText = "";
+
+        public MaterialPropertyReorderableList(SerializedObject serializedObject, SerializedProperty elements, string filter) : base(serializedObject, elements, draggable:false, displayHeader:true, displayAddButton:false, displayRemoveButton:false)
+        {
+            this.drawElementCallback = DrawElement;
+            this.elementHeightCallback = ElementHeight;
+            this.drawHeaderCallback = DrawHeader;
+            _filterText = filter;
+        }
+
+        private void DrawHeader(Rect rect)
+        {
+            var cursor = rect;
```

Comment:
This function has duplicate lines with some code in `DrawElement` (0.45, 0.1,0.45 ratio, for example).
Can you please wrap the duplicate code in a function ?